app/browser-sync.ts

```diff
@@ -83,6 +83,29 @@ export class BrowserSync {
 
     return result;
   }
+
+  forkNewDriverInstance(useSameUrl?: boolean, copyMockModules?: boolean, copyConfigUpdates?: boolean) {
+    const newBrowser = this.getBrowser().forkNewDriverInstance(useSameUrl, copyMockModules, copyConfigUpdates);
+
+    const waitForAngularEnabled = this.waitForAngularEnabled();
+
+    //Use the runner class to reset global variables after spawning the new instance
+    const runner = new Runner(this.getProcessedConfig());
+    runner.setupGlobals_(newBrowser);
```

Comment:
i couldn't think of a use case where you want to open a new instance and not use it, so I felt like this is a better default.

The consistency argument is still good though. I am ok either way.